app/pb_kits/playbook/pb_selectable_list/_selectable_list.scss

```diff
@@ -0,0 +1,9 @@
+@import "../tokens/colors";
+
+[class^=pb_selectable_list]{
+  .pb_item_kit {
+    &:hover {
+      background-color: $bg_light;
+    }
+  }
+}
```

Comment:
The Select Kit (specifically, class `pb_select`)  is overriding the Selectable List Kit in React to give it a margin bottom. Can you add in css so that there is no margin bottom would be applied for the class `pb_selectable_list_item_kit`?

Reply:
We went and hopefully fixed the padding issue!